rust_src/src/fonts.rs

```diff
@@ -143,4 +213,12 @@ pub fn find_font(spec: LispObject, frame: LispObject) -> LispObject {
     }
 }
 
+/// Close FONT-OBJECT
```

Comment:
@nickdrozd this is a place where the comment does not match the variable. Should we name the arg `font_object`?

Reply:
Yeah, they should always match, except in cases where the argument name is a Rust keyword (eg `type`). It would be nice to avoid those cases too.

Reply:
Yeah, that was a mistake on my part.